THIRD-PARTY-NOTICES.txt

```diff
@@ -0,0 +1,34 @@
+vstest uses third-party libraries or other resources that may be
```

Comment:
Nit: 
```suggestion
VSTest uses third-party libraries or other resources that may be
```

or
```suggestion
Visual Studio Test Platform uses third-party libraries or other resources that may be
```

Reply:
Fixed, thanks. Decided to go with VSTest :)